<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入月份</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>741</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>天气情况</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最低气温</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最高气温</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>夜间风向</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>夜间风力</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>白天风力</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>473</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查看天气变化趋势</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
